SU2_CFD/src/output/CFlowCompOutput.cpp

```diff
@@ -691,3 +720,48 @@ bool CFlowCompOutput::SetUpdate_Averages(CConfig *config){
 }
 
 
+void CFlowCompOutput::SetAdditionalScreenOutput(CConfig *config){
+
+  if (config->GetFixed_CL_Mode()){
+    SetFixedCLScreenOutput(config);
+  } 
+}
+
+void CFlowCompOutput::SetFixedCLScreenOutput(CConfig *config){
+  PrintingToolbox::CTablePrinter FixedCLSummary(&cout);    
+  
+  if (fabs(historyOutput_Map["CL_DRIVER_COMMAND"].value) > 1e-16){
+    FixedCLSummary.AddColumn("Fixed CL Mode", 40); 
+    FixedCLSummary.AddColumn("Value", 30);  
```

Comment:
I moved all the output to the CFlowCompOutput. I had to make a few new functions but I tried to copy the way the WriteAdditionalFiles function works. I hope this is what you had in mind @talbring 

It prints everytime the angle of attack changes and looks like this: 
![image](https://user-images.githubusercontent.com/30271435/66816099-5956d200-eeee-11e9-80f0-5ecff74fb1bb.png)

Reply:
Yep looks good